docs-content/installing.html.md.erb

```diff
@@ -7,14 +7,12 @@ This topic describes how to install and configure AppDynamics Platform Metrics i
 
 ## <a id='overview-appd-nozzle'></a> Overview
 
-AppDynamics Nozzle aggregates and forwards logs and metrics from the Loggregator Firehose to the AppDynamics Controller. The Controller process that data and provides the performance metrics of the underlying infrastructure of apps running on PCF. You can view them in the Metric Browser corresponding to the Nozzle application.
+AppDynamics Nozzle is a Cloud Foundry (CF) component dedicated to aggregating and forwarding logs and metrics from the **Loggregator [Firehose](https://docs.cloudfoundry.org/loggregator/architecture.html#firehose)** to the AppDynamics Controller. The Controller processes the data stream including logs, HTTP events, and container metrics from all applications running on the underlying infrastructure on PCF. You can view them in the Metric Browser corresponding to the Nozzle application.
 
-When you successfully configure AppDynamics Nozzle on PCF, a default application named appd-nozzle, is created on the AppDynamics Controller. Additionally, a default tier named appd-nozzle-tier and a default node named appd-nozzle-node are created under which corresponding metrics are recorded.
+When you successfully configure AppDynamics Nozzle on PCF, a default application named **appd-nozzle**, is created on the AppDynamics Controller. Additionally, a default tier named **appd-nozzle-tier** and a default node named **appd-nozzle-node**  are created under which corresponding metrics are recorded.
 
 ## <a id='before-you-start'></a> Before You Start
```

Comment:
In the Before You Start section, we say:
Acquire at least one Go SDK Agent License.
I am unclear if this license has to be acquired through AppDynamics, Go/Google, or Pivotal Cloud Foundry. Can we specify, maybe with a link, on how to proceed?

Reply:
It is an AppDynamics License. Perhaps we should rephrase it as AppDynamics Go SDK Agent License.